### DISCUSSION TITLE
Make OpenShell larger on 4K Display?

### DISCUSSION THREAD

**Discussion by andyxo-xo** [Open-Shell/Open-Shell-Menu#1745] [General]
Hi guys, simple question please. Am on W10. Is there a way to make the Open-Shell Start Menu appear larger when I left click the start button on my 85" 4K TV (Extended Display 2) without having to change any other W10 settings? On my 1080 monitor the Open-Shell start menu is perfect size but on my 4K TV it's ummm just a little bit small. It's not too bad but being able to make it about 33% larger would be about perfect. 

So to make it a little bit bigger when opened on the 4K Display 2 is there any kind of Open-Shell DPI setting I should know about or could play with? I already have the 4K Extended Display 2 set for 200% scaling in W10 and don't want to mess with that as that seems pretty good. Thanks fellas :-)

**Comment by ge0rdi** [comment]
There is `Override system DPI` option. So you can try to change that.

> I already have the 4K Extended Display 2 set for 200% scaling in W10

`Open-Shell` is supposed to scale according to scaling set in Windows display settings.
So even with 200% scaling it appears small?

**Reply by andyxo-xo** [reply]
Hi, thanks for your reply. Which .exe exactly would I change DPI settings? I played around with C:\Program Files\Open-Shell\\ **StartMenu.exe** but no effect :-(

Actually I would like to amend my original question a little bit if I may please. With W10's "Show taskbar on all displays" ON I think OpenShell's Start Menu is indeed scaling 200%, still looks a tad small, but it's still readily useable so I should not complain :-)

However, when used in conjunction with DisplayFusion's own "Taskbar Mode; All taskbars show on all windows" ON and W10 "Show taskbar on all displays" OFF, OpenShell's Start Menu is now so very tiny, is not being scaled 200%, and can barely even see it on my 85" 4K TV. I think this is the crux of the problem and why I would like to independently size OpenShell Start Menu larger so it works with DisplayFusion's Taskbar Mode ON and not use W10 "Show taskbar on all displays". If you ask why it's because with DisplayFusion controlling my extended display taskbar I can still get all my SystemTray icons but with W10 controlling that extended display taskbar I don't.

I hope I explained this better :-)

**Reply by ge0rdi** [reply]
You should run `Open-Shell Menu Settings` shortcut from start menu.
And then search for `Override system DPI` setting.

**Reply by andyxo-xo** [reply]
Thanks again for your reply. I found that setting and yes setting that "Override system DPI" to 200 sure fixed my problem on the 85" 4K Extended Display. But now on my 1080 primary display it's totally gigantic. Is there no way to differentiate the two start menus like to make my primary display 1080 a normal sized open-shell start menu while on the 4K display the open-shell start menu DPI is set to 200 (bigger) so I can see it? Cheers again...

**Reply by ge0rdi** [reply]
Right, the setting does change DPI globally so it affects all monitors. Sorry, didn't realize that.

Now that I'm going through code, I think we don't even use per-monitor DPI in start menu (as I thought for some reason).
I guess that is something that could be improved in the future.

**Reply by ge0rdi** [reply]
> Now that I'm going through code, I think we don't even use per-monitor DPI in start menu (as I thought for some reason).

Hmm, ok, it seems we do get DPI of monitor where taskbar is located when Start menu is triggered for that taskbar.
So if there are multiple monitors (and taskbars) we should obtain DPI/scaling for the one where Start menu is displayed.

I guess this should work. And I think you wrote it works that way with standard Windows taskbar.
But it doesn't work with `DisplayFusion` taskbar?
In such case I guess I cannot help much. I don't have multi-monitor setup at hand (atm), nor am I familiar with DisplayFusion.

So I'm afraid that right now the only option you have is to use standard Windows taskbar (or use monitors with the same DPI).

**Reply by andyxo-xo** [reply]
Thanks very much for your responses :-) And ya, if W10 would show the system tray on extended monitors then I wouldn't be having any of these problems. I could use open-shell normally without using DisplayFusion... darn
